features/tests.feature

```diff
@@ -24,15 +24,31 @@ Feature: The tests command
     Then the output should contain "# Running features"
       And the output should contain "> Only: features/some.feature"
 
+  Scenario: Run only tests in a directory
+    Given an empty file named "spec/spec_helper.rb"
+
+    When I run `geordi tests spec`
+    Then the output should contain "# Running specs"
+      And the output should contain "Only: spec"
+
+    When I run `geordi tests features`
+    Then the output should contain "# Running features"
+      And the output should contain "> Only: features"
+
   Scenario: Geordi's options are processed
     Given an empty file named "features/some.feature"
 
     When I run `geordi tests features/some.feature -v`
     Then the output should contain "> bundle exec cucumber"
 
+    When I run `geordi tests features/some.feature`
+    Then the output should not contain "> bundle exec cucumber"
+
   Scenario: Unknown options are passed through
     Given an empty file named "spec/spec_helper.rb"
 
     When I run `geordi tests spec/ -c`
     Then the output should contain "Util.run! bundle exec rspec spec/ -c"
 
+    When I run `geordi tests spec/`
+    Then the output should not contain "Util.run! bundle exec rspec spec/ -c"
```

Comment:
I think this doesn't make it any clearer. If I don't type `-c` I'd be pretty surprised to see it in the output. Is the Feature still green if the step is
```
Then the output should not contain "Util.run! bundle exec rspec spec/"
``` 
?
Otherwise, is `-c` a valid command line option for rspec? If so what does it do? i couldn't find any information about it.  
Maybe you can find an example where `geordi tests spec/ --some-option` produces an output that is different from running `geordi tests spec/ ` in other regards than the  `-some-option` missing.

Reply:
`geordi rspec` doesn't do anything to the arguments, it just appends them if they are there. As rspec never actually runs in tests, the only difference in the output is the presence or absence of the option.

Reply:
Okay. I expected aruba to actually run the rspec command and give us its output. But as it is, I think you can drop these two lines again. As I said before, if I don't type `-c` I'd be pretty surprised to see it in the output. Sorry for the back and forth, I just thought there's an easy way to make the scenario more conclusive.